README.md

```diff
@@ -230,11 +230,13 @@ User agents need not perform this normalization on the domains in their static l
 # Clearing Site Data on Set Transitions
 Sites can change which First-Party Set they are a member of. We need to pay attention to these transitions so that they don’t link user identities across all the FPSs they’ve historically been in. In particular, we must ensure that a domain cannot transfer a user identifier from one First-Party Set to another when it changes its set membership.
 
-In order to achieve this, site data needs to be cleared on certain transitions. The clearing should behave like [`Clear-Site-Data: "*"`](https://www.w3.org/TR/clear-site-data/#grammardef-), which includes cookies, storage, cache, as well as execution contexts (documents, workers, etc.). We don’t differentiate between different types of site data because:
+In order to achieve this, all site data needs to be cleared on certain transitions, including cookies, storage, cache, as well as execution contexts (documents, workers, etc.). We don’t differentiate between different types of site data because:
```

Comment:
What "all site data" means is rather unclear, especially given the suggestion below about it being across partitions.

Reply:
Thanks for the input! Content is updated accordingly.